pages/linux/dmenu.md

```diff
@@ -2,6 +2,7 @@
 
 > Dynamic menu.
 > Creates a menu from a text input with each item on a new line.
+> More information: <https://dwm.suckless.org/>.
```

Comment:
```suggestion
> More information: <https://manned.org/dmenu>.
```
Even though it's the official website, I couldn't find any info on how to use the command.